docs/guides/migrating-from-camunda-platform-7/conceptual-differences.md

```diff
@@ -0,0 +1,149 @@
+
+The implications for your process solution and the programming model are described below. Conceptually, the only really big difference is that with a remote engine, you cannot share technical [ACID transactions](https://en.wikipedia.org/wiki/ACID) between your code and the workflow engine. You can read more about it in the blog post ["Achieving consistency without transaction managers"](https://blog.bernd-ruecker.com/achieving-consistency-without-transaction-managers-7cb480bd08c).
+
+#### Different data types
+
+In Camunda Platform 7, you can store different data types, including serialized Java objects.
+
+Camunda Platform 8 only allows storage of primary data types or JSON as process variables. This might require some additional data mapping in your code when you set or get process variables.
+
+Camunda Platform 7 provides [Camunda Spin](https://docs.camunda.org/manual/latest/reference/spin/) to ease XML and JSON handling. This is not available with Camunda Platform 8, and ideally you migrate to an own data transformation logic you can fully control (e.g. using Jackson).
+
+To migrate existing process solutions that use Camunda Spin heavily, you can still add the Camunda Spin library to your application itself and use its API to do the same data transformations as before in your application code.
+
+#### Expression language
+
+Camunda Platform 7 uses [JUEL (Java Unified Expression Language)](https://docs.camunda.org/manual/latest/user-guide/process-engine/expression-language/) as the expression language. In the embedded engine scenario, expressions can even read into beans (Java object instances) in the application.
+
+Camunda Platform 8 uses [FEEL (Friendly-Enough Expression Language](/components/modeler/feel/what-is-feel.md) and expressions can only access the process instance data and variables.
+
+Most expressions can be converted (see [this community extension](https://github.com/camunda-community-hub/camunda-7-to-8-migration/blob/main/backend-diagram-converter/core/src/main/java/org/camunda/community/migration/converter/expression/ExpressionTransformer.java) as a starting point), some might need to be completely rewritten, and some might require an additional service task to prepare necessary data (which may have been calculated on the fly when using Camunda Platform 7).
+
+#### Different Connector infrastructure
+
+Camunda Platform 7 provides several [Connectors](https://docs.camunda.org/manual/latest/reference/connect/). Camunda Platform 8 now also offers multiple [Connectors](/components/connectors/out-of-the-box-connectors/available-connectors-overview.md) as well.
+
+To migrate existing Connectors, create a small bridging layer to invoke these Connectors via a custom [job workers](/components/concepts/job-workers.md).
```

Comment:
for ootb connectors, you could also leverage the C8 Rest Connector. For Custom Connectors a bridging layer is a good idea.